src/Common/REST/TEC/V1/Endpoints/OpenApiDocs.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * OpenAPI docs endpoint.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\REST\TEC\V1\Endpoints
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Common\REST\TEC\V1\Endpoints;
+
+// phpcs:disable StellarWP.Classes.ValidClassName.NotSnakeCase
```

Comment:
Why? Does it _need_ to be in CamelCase?

Reply:
Yes, if you visit https://www.openapis.org/ the OpenAPI is a name and its written like that. The `Docs` part of the name could be snake case but i'd have to ignore in any case.

Reply:
OK, I figured it was a branding thing.